content/tutorials/manage-account-settings/disable-master-key-pair.md

```diff
@@ -0,0 +1,497 @@
+# Disable Master Key Pair
+
+This page describes how to disable the [master key pair](cryptographic-keys.html) that is mathematically associated with an [account](accounts.html)'s address. You may wish to do this if you believe your account's master key pair has been compromised, or if you want to make [multi-signing](multi-signing.html) the _only_ way to submit transactions from your account.
+
+**Warning:** Disabling the master key pair removes one method of [authorizing transactions](transaction-basics.html#authorizing-transactions). You should be sure you can use one of the other ways of authorizing transactions, such as with a regular key or by multi-signing, before you disable the master key pair. (For example, if you [assigned a regular key pair](assign-a-regular-key-pair.html), make sure that you can successfully submit transactions with that regular key.) Due to the decentralized nature of the XRP Ledger, no one can restore access to your account if you cannot use the remaining ways of authorizing transactions.
+
+**To disable the master key pair, you must use the master key pair.** However, you can _re-enable_ the master key pair using any other method of authorizing transactions.
+
+## Prerequisites
+
+To disable the master key pair for an account, you must meet the following prerequisites:
+
+- You must have an XRP Ledger [account](accounts.html) and you must be able to sign and submit transactions from that account using the master key pair. See also: [Set Up Secure Signing](set-up-secure-signing.html).
+    - Generally, you need to know account's master seed value to sign this transaction. A seed value is commonly represented as a [base58][] value starting with "s", such as `sn3nxiW7v8KXzPzAqzyHXbSSKNuN9`. A notable exception is that a [dedicated signing device](set-up-secure-signing.html#use-a-dedicated-signing-device) stores the seed value securely so you don't need to know it.
+- Your account must have at least one method of authorizing transactions other than the master key pair. In other words, you must do one or both of the following:
```

Comment:
you must do one or both of the following: - technically it is possible to disable the master-key pair and lose your value, if you haven't done this, right? I know you discuss this elsewhere, but because it's so significant, I'd rephrase this to indicate that if you don't do one of the following two things, you will lose your value.

Reply:
No, it is not possible. The XRP Ledger does not allow a transaction to remove the last method of authorizing transactions from an account.

If you have neither assigned a regular key pair or set up multi-signing, the AccountSet to disable the master key pair fails with `tecNO_ALTERNATIVE_KEY`. I've added an extra reminder about this to the stage in the tutorial where that error would come up.